include/pelib/PeHeader.h

```diff
@@ -1235,16 +1235,20 @@ namespace PeLib
 		// Verify the NT signature
 		if (header.Signature != PeLib::PELIB_IMAGE_NT_SIGNATURE)
 			setLoaderError(LDR_ERROR_NO_NT_SIGNATURE);
-		if (header.FileHeader.Machine == 0 || header.FileHeader.SizeOfOptionalHeader == 0)
+
+		// Zoo\PE-culiarities\ok_but_total_fuckups\SizeOfOptionalHeader_Zero\unpacked.ex_ 
```

Comment:
Is this sample publicly available (e.g. in regression tests)? We should reference samples either by their SHA-256 hash, or by path in the retdec-regression-tests repository. Use of internal paths is meaningless for other people.

Reply:
Internal path removed in commit 9ccd3943c1a0917ffdaff5ab983fa7cdbb6dc5a2.